tsdb/chunks/head_chunks_test.go

```diff
@@ -231,15 +237,18 @@ func TestChunkDiskMapper_Truncate(t *testing.T) {
 
 	// Truncating files after restart.
 	require.NoError(t, hrw.Truncate(sixthFileMinT))
-	verifyFiles([]int{6, 7, 8, 9, 10})
```

Comment:
We still need to verify that there is only `6,7,8,9` around. Because only the creation of new file has to wait for new chunk, not the truncation of old files.

Reply:
you're right, added that in 4d3d59494f79d57c0faa143668544adbedc81f97